spec/lib/droplet_kit/resources/project_resource_spec.rb

```diff
@@ -97,12 +97,11 @@
 
   describe '#list_resources' do
```

Comment:
Please use the [paginated index](https://github.com/digitalocean/droplet_kit/blob/d5e166cc8e2047041f88383a8c38f5717d75c86a/spec/support/shared_examples/paginated_endpoint.rb) shared example, so any changes are reflected in all related tests. See the [droplet spec](https://github.com/digitalocean/droplet_kit/blob/d5e166cc8e2047041f88383a8c38f5717d75c86a/spec/lib/droplet_kit/resources/droplet_resource_spec.rb#L89)